docs/blocks/data_loading.md

```diff
@@ -0,0 +1,993 @@
+```
+
+To learn more about using configuration settings, see [Configuration Settings](#configuration-settings).
+
+Then the following code uses the functions:
+
+-   `execute()` - executes an arbitrary query on your data warehouse. In this case, the warehouse, database, and schema to use are selected.
+-   `load()` - loads the results of a `SELECT` query into a Pandas DataFrame.
+-   `export()` - stores data frame as a table in your data warehouse. If the table exists, then the data is appended by default (and can be configured with other behavior, see [Snowflake](#snowflake) API). If the table doesn't exist, then the table is created with the given schema name and table name (the table data types are inferred from the Python data type).
+
+```python
+with loader:
+    loader.execute('USE WAREHOUSE my_warehouse;')
+    loader.execute('USE DATABASE my_database;')
+    loader.execute('USE SCHEMA my_schema;')
+    df = loader.load('SELECT * FROM test_table;')
+    loader.export(df, 'my_schema', 'test_table')
+```
+
+The `loader` object manages a direct connection to your Snowflake data warehouse, so it is important to make sure that your connection is closed once your operations are completed. You can manually use `loader.open()` and `loader.close()` to open and close the connection to your data warehouse or automatically manage the connection with a context manager.
+
+To learn more about loading data from Snowflake, see the [Snowflake](#snowflake) API for more details on member functions and usage.
+
+# Client APIs
+
+This section covers the API for using the following data loaders.
+
+## Redshift
+
+`mage_ai.io.redshift.Redshift`
+
+Handles data transfer between a Redshift cluster and the Mage app. Mage uses temporary credentials to authenticate access to a Redshift cluster. There are two ways to specify these credentials:
+
+1. Pre-generate temporary credentials and specify them in the configuration settings. Add the following keys to the configuration settings for `Redshift` to use the temporary credentials:
+    ```yaml
+    REDSHIFT_DBNAME: Name of Redshift database to connect to
+    REDSHIFT_HOST: Redshift Cluster hostname
+    REDSHIFT_PORT: Redshift Cluster port. Optional, defaults to 5439
+    REDSHIFT_TEMP_CRED_USER: Redshift temp credentials username
+    REDSHIFT_TEMP_CRED_PASSWORD: Redshift temp credentials password
+    ```
+2. Provide an IAM Profile to automatically generate temporary credentials for connection. The IAM profile is read from `~/.aws/` and is used with the `GetClusterCredentials` endpoint to generate temporary credentials. Add the following keys to the configuration settings for `Redshift` to generate temporary credentials
+   `yaml REDSHIFT_DBNAME: Name of Redshift database to connect toName of Redshift database to connect to REDSHIFT_DBUSER: Redshift database user to generate credentials for REDSHIFT_CLUSTER_ID: Redshift cluster ID REDSHIFT_IAM_PROFILE: Name of the IAM profile to generate temp credentials with `
```

Comment:
<img width="1092" alt="image" src="https://user-images.githubusercontent.com/80284865/180891083-86b3782b-92ab-4425-99af-2336b29aedeb.png">
format is off